its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -244,6 +244,7 @@ private void assertLineCountForProjectUnderTest(String projectKey) {
 
   @Test
   void testExcludedAndTest_AnalyzeTestProject() throws Exception {
```

Comment:
Those 3 tests started failing when pinning the plugin version (on SQ8.9 tests too, for some reason)

Reply:
That's odd and unexpected. Can you find out why?